src/main/java/com/baidu/hugegraph/api/graph/VertexAPI.java

```diff
@@ -74,6 +75,15 @@ public Vertex create(Vertex vertex) {
         return result.readList(this.type(), Vertex.class);
     }
 
+    public int update(BatchOlapPropertyRequest request) {
+        this.client.checkApiVersion("0.56", "batch property update");
+        MultivaluedHashMap<String, Object> headers = new MultivaluedHashMap<>();
+        headers.putSingle("Content-Encoding", BATCH_ENCODING);
+        String path = String.join("/", this.path(), "olap/batch");
```

Comment:
is there olap property single update

Reply:
is there olap property single update?

Reply:
there is no single update in server api so far. need to add one?